godtools/App/Services/Renderer/Views/MobileContent/Views/ContentButton/MobileContentButtonViewModel.swift

```diff
@@ -38,29 +38,33 @@ class MobileContentButtonViewModel: NSObject, MobileContentButtonViewModelType {
         
         buttonWidth = MobileContentViewWidth(dimension: buttonModel.width)
         
-        let buttonColor: UIColor = buttonModel.buttonColor
-        let buttonTitleColor: UIColor? = buttonModel.text?.textColor
-                
+        let defaultBackgroundColor: UIColor = buttonModel.buttonColor
+        let defaultTitleColor: UIColor = buttonModel.text?.textColor ?? renderedPageContext.pageModel.primaryTextColor
```

Comment:
hopefully my work on making `text` non-null will make this fallback behavior unnecessary

Reply:
Hey @frett I like the sound of that.

Reply:
https://github.com/CruGlobal/kotlin-mpp-godtools-tool-parser/pull/228

Reply:
Hey @frett I'm going to create a separate ticket for this one too.

Reply:
https://jira.cru.org/browse/GT-1481